resources/js/alpine/linkifyDirective/linkifyDirective.ts

```diff
@@ -0,0 +1,20 @@
+import linkifyHtml from 'linkify-html';
+import type { DirectiveData, DirectiveUtilities } from 'alpinejs';
+
+export function linkifyDirective(
+  el: HTMLElement,
+  _: DirectiveData,
+  { effect }: DirectiveUtilities,
+) {
+  effect(() => {
+    el.innerHTML = linkifyHtml(el.innerHTML, {
+      formatHref: {
+        url: (href) => `/redirect?url=${href}`,
```

Comment:
redirect is misleading when the link is pointing at retroachievements.org: http://localhost:64000/user/TheUnwanted

![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/de3798b0-6283-4a77-a012-b4d4d372b97f)

http://localhost:64000/user/TheJoh4nx
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/5463ec2f-d913-43d6-94f8-ec4bc6bb4812)

Reply:
Good catch. In latest, internal links will no longer pass through the redirect gateway.